cmd/bmp180/main.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// bmp180 reads the current temperature and pressure from a BMP180.
```

Comment:
Oops sorry I was really confused, ignore my comment about package name.

It kind of suck that the official one is of less quality. I'm not a fan of linking to adafruit's copies.